# Protocal prefix
string VEL = "V"
string POS = "P"
string ENC = "E"
string LIMIT = "L"

std_msgs/Header header

string predicate
float64[] data